### PR TITLE
Handlebars

### PR DESCRIPTION
**Pull request by brendandahl**

### CODE REVIEW

File: gulpfile.babel.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const plugins = loadPlugins({
   lazy: false,
 });
 
-import babelRegister from 'babel/register';
+import babelRegister from 'babel-core/register';
```
Comment (digitarald) [inline gulpfile.babel.js]: [babel docs](https://babeljs.io/docs/usage/require/) suggest otherwise, any idea where the disconnect comes from?

Comment (brendandahl) [inline gulpfile.babel.js]: I can't find any docs on it, but it looks like babel is the cli and babel-core will just be the compiler parts.

Comment (brendandahl) [inline gulpfile.babel.js]: If we do use babel we need to do npm install -g babel

Comment (digitarald) [inline gulpfile.babel.js]: Lets keep it in there if this fixes babel for you and revisit later

**Pull request merged**